spec/controllers/sponsors_controller_spec.rb

```diff
@@ -133,7 +133,8 @@
         name: "Ted Berry",
         email: "ted@example.com",
         postcode: "G34 0BX",
-        location_code: "GB-SCT"
+        location_code: "GB-SCT",
+        privacy_notice: "1"
```

Comment:
Maybe add this to the factory with a default of "1" to save changing all the specs

Reply:
Sorry again - thought this was a FactoryBot call